test/bh/content-test.js

```diff
@@ -0,0 +1,85 @@
+var fixtures = require('../fixtures')('bemhtml');
+var compile = fixtures.compile;
+
+describe('content()(×)', function() {
+  it('should return bemjson content', function() {
+    compile(function() {
+      block('button').def()(function() {
+        return this.ctx.content;
+      });
+    })
+      .apply({ block: 'button', content: 'Hello' })
+      .should.equal('Hello');
+  });
+  it('should set bemjson content', function() {
+    compile(function() {
+      block('button').content()({ elem: 'text' });
+    })
+      .apply({ block: 'button' })
+      .should.equal(
+        '<div class="button"><div class="button__text"></div></div>');
+  });
+  it('should set bemjson array content', function() {
+    compile(function() {
+      block('button').content()([ { elem: 'text1' }, { elem: 'text2' } ]);
```

Comment:
Can you replace array literals by function generators in all tests? We need to show only good examples.